app/components/speaker-list.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+const { Component } = Ember;
+
+export default Component.extend({
+  speakers: [{  name: 'Speaker 1', organisation: 'Org 1' },
```

Comment:
The speakers data should be passed in from the parent as a parameter to `speaker-list` component. Also, add social links to the dummy object model.